main.go

```diff
@@ -216,6 +224,20 @@ func submarinerClusterFrom(submSpec *types.SubmarinerSpecification) types.Submar
 	}
 }
 
+func startHttpServer() *http.Server {
+	srv := &http.Server{Addr: ":8080"}
+
+	http.Handle("/metrics", promhttp.Handler())
+
+	go func() {
+		if err := srv.ListenAndServe(); err != http.ErrServerClosed {
+			klog.Fatalf("Error starting metrics server: %v", err)
```

Comment:
We should call `fatal` to set the Gateway status.

Reply:
Or perhaps just log the error and continue running — the gateway can run without metrics...